conftest.py

```diff
@@ -37,6 +37,10 @@
 
 log = logging.getLogger(__name__)
 
+# The following environment variable allows for log collection to be turned off.
+# This is useful, for exampl in testing.
+INTEGRATION_TEST__SKIP_LOG_COLLECTION = os.environ.get('INTEGRATION_TEST__SKIP_LOG_COLLECTION', False)
```

Comment:
- How about INTEGRATION_TEST_LOG_COLLECTION that defaults to true? Having a boolean value that disables something can get confusing
- Any reason for the double underscore?
- Would probably need to do something like this, as the os.environ value will be a string if set: `bool(os.environ.get('name', 'True'))`

Reply:
I didn't want to inadvertently break something in the test, and thus went for the not False approach. I'll make the change though.

The double underscore is from my time with Airflow... Thinking on it now, it's not our norm. Will change.